content/cli/users.md

```diff
@@ -93,30 +92,28 @@ New password set
 #### Usage 
 `cfy users set-role [OPTIONS] USERNAME`
 
-Set a role for a specific user. <br>
+Set the system-wide (security) role for a specific user. <br>
 
 `USERNAME` is the username of the user
 
-Users are created with the default system-wide `default` role. This command enables you to change a user's role to a Cloudify Manager administrator.
+the system-wide role of a user represents whether the user is a system admin on the manager, or not. To give the user sys-admin permissions, set this role to 'sys_admin'. Otherwise, the user should have the system-wide role 'default', which indicates that the user needs to be explicitly assigned to tenants to access their resources, 
```

Comment:
The system-wide role defines the user as a 'sys-admin' (admin user) or 'default' (non-admin user). To give the user sys-admin permissions, set this role to 'sys_admin'. Otherwise, the user has the system-wide role 'default'. A 'default' user must be explicitly assigned to tenants in order to perform actions and access resources.